src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDownForced.groovy

```diff
@@ -16,8 +16,20 @@ class ComposeDownForced extends DefaultTask {
 
     @TaskAction
     void down() {
+
+        def dependentServices = []
+        if(settings.removeDependents) {
+            if(settings.startedServices)
+            {
+                dependentServices = settings.composeExecutor.getDependentServices(
+                        settings.startedServices).toList()
+            }
+        }
+
+        def servicesToStop = [*settings.startedServices, *dependentServices]
```

Comment:
This is probably just cosmetics - it would be great to uniquize services names here. Not sure if repeated services names would cause a trable in `docker-compose` commands.

Reply:
👍